incident.go

```diff
@@ -659,7 +659,7 @@ type ResponderRequestResponse struct {
 // ResponderRequestTarget specifies an individual target for the responder request.
 type ResponderRequestTarget struct {
 	APIObject
-	Responders IncidentResponders `json:"incident_responders,omitempty"`
+	Responders []IncidentResponders `json:"incidents_responders,omitempty"`
```

Comment:
I did consider if the `IncidentResponders` type should be renamed `IncidentResponder` but decided to keep this PR to the minimum diff and not to break the interface